src/Robo/Plugin/Traits/DrupalVersionTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Usher\Robo\Plugin\Traits;
+
+/**
+ * Trait to determine what version of Drupal is being run.
+ */
+trait DrupalVersionTrait
+{
+    /**
+     * Check if version of Drupal in question is Drupal 7.
+     *
+     * @param string $drupalRootPath
+     *   The path to the Drupal root.
+     */
+    protected function drupalVersionIsD7(string $drupalRootPath): bool
+    {
+        // @see https://git.drupalcode.org/project/drupal/-/blob/7.x/includes/bootstrap.inc
+        $bootstrapPath = "$drupalRootPath/includes/bootstrap.inc";
+        if (file_exists($bootstrapPath)) {
+            include_once($bootstrapPath);
+            if (substr(VERSION, 0, 2) === '7.') {
```

Comment:
@adamzimmermann Ahh. PHPStan really doesn't like the use of this constant. I suppose I understand.

<img width="651" alt="CleanShot 2023-04-21 at 10 39 47@2x" src="https://user-images.githubusercontent.com/20355/233664588-a158696b-35c3-4069-93d7-462f22474755.png">